.github/ISSUE_TEMPLATE/issue-default.yml

```diff
@@ -31,7 +31,7 @@ body:
     attributes:
       label: Links to other issues
       description: |
-        "Add issue #numbers this relates to and how (e.g., 🚧 :construction: Blocks, ⛔️ :no_entry: Is blocked by, 🔄 :repeat: Relates to)."
+        "Add issue #numbers this relates to and how (e.g., 🚧 [construction] Blocks, ⛔️ [no_entry] Is blocked by, 🔄 [arrows_counterclockwise] Relates to)."
```

Comment:
I think this square bracket notation w/out a link may have been broken in older github versions, but now it renders normally. If this still renders you'd need to use an escape character so like \[construction\] but I don't think its an issue. I tested it in a markdown file and it worked so I honestly don't see why it wouldn't here

Reply:
Square brackets also worked for me. We'll find out when it's merged whether it actually works on GitHub!